sdk/core/azure-core/azure/core/tracing/decorator.py

```diff
@@ -36,38 +36,48 @@
     TYPE_CHECKING = False
 
 if TYPE_CHECKING:
-    from typing import Callable, Any
+    from typing import Callable, Dict, Optional, Any, cast
 
 
-def distributed_trace(func=None, name_of_span=None):
-    # type: (Callable, str) -> Callable[[Any], Any]
+def distributed_trace(_func=None, name_of_span=None, tracing_attributes=None):
```

Comment:
keyword only argument? I don't expect there to be any logical ordering of any additional parameters we may add in the future...

Reply:
Fixed